ui/src/data-services/hooks/projects/utils.ts

```diff
@@ -13,5 +13,9 @@ export const convertToServerFormData = (fieldValues: any) => {
     data.append('image', '')
   }
 
+  if (fieldValues.openSource) {
+    data.append('license', 'open-source')
```

Comment:
Let's tweak this, if backend prefers the license to be passed in a different way.